main.go

```diff
@@ -2,11 +2,15 @@ package main
 
 import (
 	"log"
+	"os"
 
 	"github.com/theredditbandit/pman/cmd"
 )
 
 func main() {
 	log.SetFlags(log.Lshortfile | log.LstdFlags)
-	cmd.Execute()
+	err := cmd.Execute()
+	if err != nil {
+		os.Exit(1)
```

Comment:
The error returned by the sub command are already printed before the Usage

https://github.com/spf13/cobra/blob/5a1acea3210649f3d70002818ec04b09f6347062/command.go#L1099-L1102

You can test by your own.

I tried by priniting in the main, but then the error is displayed twice

```
Error:
Usage
Error
```

I'm unsure why cobra has this behavior

I also did this, because when you use SilenceUsage

https://github.com/spf13/cobra/blob/5a1acea3210649f3d70002818ec04b09f6347062/command.go#L1134-L1136

The error would be printed twice

```
Error:
Usage
Error
```

That's why I didn't print error here, and yes, it's strange.

I can add a comment, maybe

Reply:
you're right . cobra prints out the error messages automatically. I had no idea 

This feels quite unintuitive.

Reply:
Definitely !

Reply:
> Definitely !

About the fact it's counterintuitive, not about the fact I was right, 😂😅🤣😂

Reply:
😂😂